### PR TITLE
[@types/node]: update directories for node v22

### PR DESCRIPTION
**Pull request by danielcuque**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### Motivation
[Node 22](https://nodejs.org/en/blog/announcements/v22-release-announce)

### CODE REVIEW

File: types/webpack-manifest-plugin/webpack-manifest-plugin-tests.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ const options: Options = {
         return file;
     },
     publicPath: "prod",
-    removeKeyHash: /[a-Z0-9]/g,
+    removeKeyHash: /[A-Za-z0-9]/g,
```
Comment (Semigradsky) [inline types/webpack-manifest-plugin/webpack-manifest-plugin-tests.ts:18]: How this change related to node v22?

**Comment by Semigradsky** [comment]
Would be nice to merge https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69212 before this...

File: types/webpack-manifest-plugin/webpack-manifest-plugin-tests.ts (hunk 1 above)
Comment (danielcuque) [reply types/webpack-manifest-plugin/webpack-manifest-plugin-tests.ts]: I'm not sure why the tests are failing, so I tried to fix it

**Comment by codershiba** [comment]
#69212 is now merged, work on this PR can be resumed. Try closing and reopening the PR to fix the CI error.

**Pull request closed without merging**